arbor-robotics/steward
# Milestone pr1

Added MVPs for:
- Forest Plan generation
- Routing
- Motion control

Also:
- added to documentation (https://arbor-robotics.github.io/steward)
- cleaned up and added new ROS params
- cleaned up the launch file
- remove *.pyc files

To replicate:
1. Run ArborSim using the latest version
2. Run Steward like:
``` bash
$ cd steward
$ colcon build --symlink-install
$ . install/setup.bash
$ ros2 launch sim.launch.py
```
3. Run Rviz in a new terminal: `$ rviz2`.

You should see something like this: https://youtu.be/MP3kJ36SKxM?si=2xzJz1AbkYruGoFG
